clients/src/test/java/org/apache/kafka/clients/consumer/ConsumerRecordTest.java

```diff
@@ -49,6 +50,7 @@ public void testShortConstructor() {
         assertEquals(ConsumerRecord.NULL_SIZE, record.serializedValueSize());
         assertEquals(Optional.empty(), record.leaderEpoch());
         assertEquals(Optional.empty(), record.deliveryCount());
+        assertNotNull(record.leaderEpoch());
```

Comment:
`leaderEpoch` is `Optional<Integer>`, I think the `assertNotnull` isn't a good assertion in here, and `record.leaderEpoch()` already check at line 51.